client/main.lua

```diff
@@ -770,6 +772,57 @@ AddEventHandler('esx_mechanicjob:hasExitedMarker', function(zone)
 	ESX.UI.Menu.CloseAll()
 end)
 
+--VehScripts
+AddEventHandler('esx_mechanicjob:hasEnteredvehMarker', function(station, part, partNum)
```

Comment:
instead of creating a duplicate EventHandler, just use the current esx_mechanicjob:hasEnteredMarker just, add the required arguments on the function.